src/flipclock/js/libs/time.js

```diff
@@ -177,6 +177,27 @@
 		},
 
 		/**
+		 * Gets a digitized weekly counter
+		 *
+		 * @return  object  Returns a digitized object
+		 */
+
+		getWeekCounter: function(includeSeconds) {
+			var digits = [
+				this.getWeeks(),
+				(this.getWeeks() + 1) * 7 - this.getDays(),
```

Comment:
I was seeing conditions where the days and week were not being calculated correctly. e.g. 11w:**7d**:10h...
where it should have been **12w:0d**:10h...
Tracked it down to line188 here. I believe this line should be:
this.getDays() - (this.getWeeks() * 7 ),